analysis/basestats/analyze.h

```diff
@@ -2,19 +2,27 @@
 
 #include "request.h"
 #include <iostream>
+#include <map>
 #include <unordered_map>
 #include <unordered_set>
+#include <vector>
 
 class Analysis {
 protected:
     std::unordered_map<std::string, int64_t > counters;
+    std::unordered_map<std::string, int64_t > ranks;
+    std::map<std::string, std::vector<Request> > reqs;
+    std::map<int64_t, int64_t > sizes;
     std::unordered_set<std::string> objids;
+    std::unordered_set<std::string> onehitwonids;
+    std::vector<int64_t> reqsizes;
 
 public:
     void processBatch(ReqBatch & batch) {
         for(auto & req: batch) {
             counters["RequestCount"]++;
             counters["TotalBytes"]+=req.size;
+            counters["TotalObjs"]++;
```

Comment:
What's the difference between counters["TotalObjs"] and counters["RequestCount"]?